### DISCUSSION TITLE
Similarity Check / Ithenticate v2

### DISCUSSION THREAD

**Discussion by eddoff** [pkp/plagiarism#47] [General]
Does this plugin work with Ithenticate v2, launched approximately a year ago?
https://www.crossref.org/blog/similarity-check-whats-new-with-ithenticate-v2/

**Comment by ctgraham** [comment]
We do not yet support API v2 as of v1.0.5-1 or the current `main` branch.  There are plans to re-write the plugin for v2.

**Reply by eddoff** [reply]
Thanks, We use the version 2 so we will be waiting for the new versopm of the plugin then.

**Comment by univie-oao-ojs** [comment]
Hello! As the most recent release v1.0.7-4 includes "Update of iThenticate API to latest version" - does this now mean that the plugin works with iThenticate v2? 
And if so: is the plugin still backwards compatible to iThenticate v1?

**Reply by touhidurabir** [reply]
> does this now mean that the plugin works with iThenticate v2?

Yes . 

> is the plugin still backwards compatible to iThenticate v1?

No, the latest released versions(for POJS/OMP/OPS 3.3 and 3.4) are compatible with iThenticate v1 API . Also note that iThenticate plan to discontinue the API v1 in future .

**Reply by univie-oao-ojs** [reply]
Thanks for your quick response!
I'm not sure if I understand the second part of your reply correctly: release v1.0.7-4 works with BOTH version of the API or should it read 'incompatible'? (And therefore unless we migrate to iThenticate v2 we should stay on release v1.0.6-x)

**Reply by touhidurabir** [reply]
> release v1.0.7-4 works with BOTH version of the API or should it read 'incompatible'?

[Plagiarism](https://github.com/pkp/plagiarism/) plugin recently have 2 new releases
version `1.0.8-1` --> [Plagiarism plugin for OJS/OMP/OPS 3.4.0-x](https://github.com/pkp/plagiarism/releases/tag/v1_0_8-1) for 3.4.0.x
version `1.0.7-4` --> [Plagiarism plugin for OJS/OMP/OPS 3.3.0-x](https://github.com/pkp/plagiarism/releases/tag/v1_0_7-4) for 3.3.0.x
as 3.3 and 3.4 have major code implementation difference . Also these 2 releases are now compatible for OJS, OMP and OPS instead of just OJS . 

> therefore unless we migrate to iThenticate v2 we should stay on release v1.0.6-x

Yes , that is the case . We do have a upgrade documentation ready , I need to check if that has been published publicly yet . Also some details are provided in the plugin's github readme doc .

**Reply by univie-oao-ojs** [reply]
Thanks! This has cleared it up now! :)
